letsencrypt/views.py

```diff
@@ -14,11 +14,9 @@
 limitations under the License.
 """
 
-from django.shortcuts import (
-    get_object_or_404,
-    render,
-)
 
+from django.shortcuts import get_object_or_404
+from django.http import HttpResponse
```

Comment:
These imports should be laid out as such (notice the alphabetic order, and the blank line spacing again:

```diff
diff --git a/letsencrypt/views.py b/letsencrypt/views.py
index 6225e18..1ce0d50 100644
--- a/letsencrypt/views.py
+++ b/letsencrypt/views.py
@@ -14,9 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 """
 
-
-from django.shortcuts import get_object_or_404
 from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+
 from .models import AcmeChallenge
```